src/mappings/encoding.ts

```diff
@@ -38,10 +38,23 @@ import {
 
 import { CurrencyId_Token as CurrencyId_TokenV10 } from "../types/v10";
 import { encodeBtcAddress, getBtcNetwork } from "./bitcoinUtils";
+import { u8aToString } from "@polkadot/util";
+import { decodeAddress, encodeAddress } from "@polkadot/util-crypto";
 
 const bitcoinNetwork: Network = getBtcNetwork(process.env.BITCOIN_NETWORK);
 const ss58format = process.env.SS58_CODEC || "substrate";
 
+const MODL_PREFIX = "modl";
+const MODL_EXCLUDE = "modldex/"
```

Comment:
I'm not sure we should be excluding this, perhaps there is also something missing [here](https://github.com/interlay/interbtc-api/pull/657).

Reply:
Let's discuss this on discord (Sander also wasn't 100% sure at the time I asked him).

Reply:
Summary from discussion on discord:
`modldex/...` accounts are considered part of circulating supply (as they can be traded by users), so squid should exclude transaction made to and from those accounts from the total system account supply value.

Or tl;dr : keep it as is.